docker-compose.yml

```diff
@@ -81,7 +81,11 @@ services:
       #- "${BAHMNI_APPS_PATH:?}/ui/app/:/usr/local/apache2/htdocs/bahmni"
       #- "${BAHMNI_APPS_PATH:?}/ui/node_modules/@bower_components/:/usr/local/apache2/htdocs/bahmni/components"
       - '${CONFIG_VOLUME:?}:/usr/local/apache2/htdocs/bahmni_config/:ro'
-
+  
+  bahmni-lab:
+    profiles: ["lab-lite","emr","bahmni-lite"]
+    image: 'bahmni/bahmni-lab:${BAHMNI_LAB_IMAGE_TAG:?}'
```

Comment:
@gokultw Can you please add this variable to .env file

Reply:
Added